numpy/ma/core.pyi

```diff
@@ -17,7 +17,7 @@ from numpy import (
 
 # TODO: Set the `bound` to something more suitable once we
 # have proper shape support
-_ShapeType = TypeVar("_ShapeType", bound=Any)
+_ShapeType = TypeVar("_ShapeType", covariant=True, bound=tuple[int, ...])
```

Comment:
Not the most interesting issue, but the parameter order of `covariant` and `bound` is exactly the opposite as the line below, which to me just feels kinda "wrong" 😛.

```suggestion
_ShapeType = TypeVar("_ShapeType", bound=tuple[int, ...], covariant=True)
```

Reply:
And here it should also be better to add a `_co` suffix to the name.